Pipelines/Templates/deploy-Solution.yml

```diff
@@ -295,4 +295,56 @@ steps:
     makerPortalUrl: '$(TestAutomationMakerPortalUrl)'
     loginMethod: '$(TestAutomationLoginMethod)'
 
+- template: Hooks\deploy-power-page-pre-hook.yml
+
+- pwsh: |
+    $portalunpackpath = "$(Build.SourcesDirectory)\$(Build.Repository.Name)\${{parameters.solutionName}}\PowerPages\"
+    Write-Host "portalunpackpath - $portalunpackpath"
+    if(Test-Path $portalunpackpath)
+    {
+        Get-ChildItem -Path $portalunpackpath -Recurse -Force        
+    }
+    else
+    {
+        Write-Host "Invalid path - $portalunpackpath"
+    }
+  displayName: "Display folder items"
+
+#- template: set-website-variable.yml
+#  parameters:
+#   serviceConnectionUrl: '${{parameters.serviceConnectionUrl}}'
+#   solutionName: '${{parameters.solutionName}}'
+
+- pwsh: |
+    $websiteName = ""
+    $solutionUnpackedFolder = "$(Build.SourcesDirectory)\$(Build.Repository.Name)\${{parameters.solutionName}}\PowerPages"
+    Write-Host "solutionUnpackedFolder - $solutionUnpackedFolder"
+    if(Test-Path $solutionUnpackedFolder)
+    {
+        $matchedFolders = Get-ChildItem $solutionUnpackedFolder | Where-Object { $_.Name -match '${{parameters.solutionName}}'}
+
+        if($matchedFolders){
+          $websiteName = $matchedFolders.Name
+        }
+        echo "##vso[task.setvariable variable=WebsiteName]$websiteName"
+    }
+    else
+    {
+       Write-Host "Invalid path - $solutionUnpackedFolder"
+    }
+    Write-Host "websiteName - $websiteName"
+  displayName: "Fetch Portal Website Name"
+
+- task: microsoft-IsvExpTools.PowerPlatform-BuildTools.upload-paportal.PowerPlatformUploadPaportal@2
+  displayName: 'Upload Power Page ${{parameters.solutionName}}'
+  inputs:
+    authenticationType: PowerPlatformSPN
+    PowerPlatformSPN: '${{parameters.serviceConnectionName}}'
+    UploadPath: '$(Build.SourcesDirectory)\$(Build.Repository.Name)\${{parameters.solutionName}}\PowerPages\$(WebsiteName)\'
+    #UploadPath: '$(Build.SourcesDirectory)\$(Build.Repository.Name)\${{parameters.solutionName}}\PowerPages\${{parameters.solutionName}}'
```

Comment:
@RajeevPentyala Remove this commented code?

Reply:
@mikefactorial Removed the code